python/ray/serve/autoscaling_metrics.py

```diff
@@ -85,6 +109,19 @@ def add_metrics_point(self, data_points: Dict[str, float], timestamp: float):
             # Using in-sort to insert while maintaining sorted ordering.
             bisect.insort(a=self.data[name], x=TimeStampedValue(timestamp, value))
 
+    def _get_datapoints(self, key: str, window_start_timestamp_s: float) -> List[float]:
+        """Get all data points given key after window_start_timestamp_s"""
+
+        datapoints = self.data[key]
+
+        idx = bisect.bisect(
+            a=datapoints,
+            x=TimeStampedValue(
+                timestamp=window_start_timestamp_s, value=0  # dummy value
```

Comment:
Can we make `TimeStampedValue`'s `value` field `Optional` instead of using a dummy value here?

Reply:
Following the old implementation `TimeStampedValue`. (this is not a change in this pr)

Reply:
Got it, we can refactor this in a follow-up change.